website/js/util.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 var months = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
-var dateFormat = (date) => { let d = new Date(date); return `${months[d.getMonth()]} ${d.getDate()}, ${d.getFullYear()}`; };
```

Comment:
I figured date display should be consistent.

I took a look at those options and couldn't decide so I opted to do this. I'd be all for moving to a library though.